recipes/invoke/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.12.2" %}
+
+package:
+  name: invoke
+  version: {{ version }}
+
+source:  
+  fn: invoke-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/invoke/invoke-{{ version }}.tar.gz
+  md5: f46192ea664325464a02f1c7f735e3cb
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  entry_points:
+    - invoke = invoke.main:program.run
+    - inv = invoke.main:program.run
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  # Python imports
+  imports:
+    - invoke
+    - invoke.parser
+    - invoke.vendor
+    - invoke.vendor.fluidity
+    - invoke.vendor.lexicon
+    - invoke.vendor.yaml2    # [py2k]
+    - invoke.vendor.yaml3    # [py3k]
+
+  commands:
+    - invoke --help
+    - inv --help
+
+about:
+  home: http://docs.pyinvoke.org
+  license: BSD-2-Clause
+  summary: 'Pythonic task execution'
+
+extra:
+  recipe-maintainers:
+    - ivoflipse
+    - Korijn
```

Comment:
Very happy! Thanks for putting so much effort into conda-forge. We've been having a hard time submitting our recipes in the past, this is brilliant!